lib/tlscontext.c

```diff
@@ -931,6 +939,14 @@ tls_context_set_ssl_options_by_name(TLSContext *self, GList *options)
         self->ssl_options |= TSO_NOTLSv11;
       else if (strcasecmp(l->data, "no-tlsv12") == 0 || strcasecmp(l->data, "no_tlsv12") == 0)
         self->ssl_options |= TSO_NOTLSv12;
+#endif
+#ifdef SSL_OP_NO_TLSv1_3
+      else if (strcasecmp(l->data, "no-tlsv11") == 0 || strcasecmp(l->data, "no_tlsv11") == 0)
+        self->ssl_options |= TSO_NOTLSv11;
+      else if (strcasecmp(l->data, "no-tlsv12") == 0 || strcasecmp(l->data, "no_tlsv12") == 0)
+        self->ssl_options |= TSO_NOTLSv12;
```

Comment:
Only the TLS1.3 check should be added.

Reply:
Thanks for the review note. I have fixed it.